### PR TITLE
Alex sandbox name unique changes for tests

### PR DESCRIPTION
**Pull request by alexzbusko**

### CODE REVIEW

File: tests/e2e/taqueria-plugin-taquito-flextesa.spec.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ const exec = utils.promisify(exec1);
 const taqueriaProjectPath = 'e2e/auto-test-taquito-flextesa-plugin';
 const contractRegex = new RegExp(/(KT1)+\w{33}?/);
 let environment: string;
-let dockerName: string = 'local';
+let dockerName: string = 'taquito-flextesa';
```
Comment (jchenche) [inline tests/e2e/taqueria-plugin-taquito-flextesa.spec.ts:17]: Would `local-taquito-flextesa` be more consistent with other sandbox names (such as `local-flextesa` and `local-tezos-client`)?

**Review by jchenche — APPROVED** [review]

**Comment by hu3man** [comment]
Hey @alexzbusko, not sure what ticket this is associated with this and only have a vague idea of what it was for

Let me know if it's something we want to keep tracking and working on, or if it's on the backburner for now

**Comment by alexzbusko** [comment]
@hu3man this is something we should do since the tests are generally run together and to make sure they won't have any collisions I want to keep this around. It is something that is on the backburner for me right now

**Comment by hu3man** [comment]
Thanks for the context @alexzbusko!

**Pull request closed without merging**